behave/helper/yapgdd/tests/data/source.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO [logs] VALUES
+(1,'CASE_CREATED',29,'2023-01-18 15:43:46.149383+00'),
```

Comment:
is the plan to update these two files with the correct times and ids next time it is run?